app/src/androidTest/java/woowacourse/movie/movie/java/SeatSelectionActivityTest.kt

```diff
@@ -30,16 +30,16 @@ import org.junit.Rule
 import org.junit.Test
 import org.junit.runner.RunWith
 import woowacourse.movie.R
-import woowacourse.movie.SeatSelectionActivity
-import woowacourse.movie.SeatSelectionActivity.Companion.DATE_KEY
-import woowacourse.movie.SeatSelectionActivity.Companion.MOVIE_KEY
-import woowacourse.movie.SeatSelectionActivity.Companion.TICKET_KEY
-import woowacourse.movie.SeatSelectionActivity.Companion.TIME_KEY
-import woowacourse.movie.TicketActivity
-import woowacourse.movie.dto.MovieDateDto
-import woowacourse.movie.dto.MovieDto
-import woowacourse.movie.dto.MovieTimeDto
-import woowacourse.movie.dto.TicketCountDto
+import woowacourse.movie.movie.activity.SeatSelectionActivity
```

Comment:
```suggestion
import woowacourse.movie.activity.SeatSelectionActivity
```
동일 패키지명으로 depth가 추가되어 있네요. 🤔